tests/plugins/test_sendmail.py

```diff
@@ -345,6 +355,12 @@ class WF(object):
             # recipients available
             assert str(ex) == 'No recipients found'
 
+        if not metadata_stored or not annotations:
+            with pytest.raises(ValueError):
+                subject, body, logs = p._render_mail(autorebuild, success,
```

Comment:
No need to store return values from a function that is expected to raise an exception.